src/shared/components/simple-select/SimpleSelect.tsx

```diff
@@ -127,7 +154,8 @@ const SimpleSelect = (
       <select
         ref={selectRef}
         className={styles.selectResetDefaults}
-        defaultValue={placeholder ? '' : undefined}
+        value={getSelectValue()}
+        onChange={selectProps.onChange ?? noop}
```

Comment:
Why do you need this line (as well as the noop)?

Reply:
I've mentioned the reason in the above comment